crates/dojo-core/src/base_test.cairo

```diff
@@ -62,3 +89,59 @@ fn test_upgrade_direct() {
     let upgradeable_dispatcher = IUpgradeableDispatcher { contract_address: base_address };
     upgradeable_dispatcher.upgrade(new_class_hash);
 }
+
+use debug::PrintTrait;
```

Comment:
```suggestion
```

Reply:
Fixed in 28a083fc7e4f823b6fa0d7f1e9f5e2585ff4af95.